test-util/src/test_data.rs

```diff
@@ -0,0 +1,101 @@
+use uuid::Uuid;
+
+pub fn unique_string(prefix: impl Into<String>) -> String {
+    format!("{}-{}", prefix.into(), Uuid::new_v4())
+}
+
+pub fn unique_cache_name() -> String {
+    unique_string("rust-sdk")
+}
+
+pub fn unique_key() -> String {
+    unique_string("key")
+}
+
+pub fn unique_value() -> String {
+    unique_string("value")
+}
+
+pub struct TestScalar {
+    pub key: String,
+    pub value: String,
+}
```

Comment:
We may need to generalize these over the `IntoBytes` type to test both bytes and strings in a coming PR.